hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -80,9 +80,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         requireNonNull(context);
         final var transactionBody = context.body().fileUpdateOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, false);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```

Comment:
These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

Reply:
fixed thanks